docs/quickstart.mdx

```diff
@@ -48,43 +34,38 @@ Let's make it happen!
 We're going to use Hardhat, but Sapphire should be compatible with your dev
 environment of choice. Let us know if things are not as expected!
 
-1. Make & enter a new directory
-2. `npx hardhat@~2.19.2 init` then create a TypeScript project.
+1. Make & enter a new directory `mkdir quickstart && cd quickstart`
+2. `npx hardhat@~2.19.2 init` then create a TypeScript project and install the project dependencies.
 3. Add [`@oasisprotocol/sapphire-hardhat`] as dependency:
 
   ```shell npm2yarn
   npm install -D @oasisprotocol/sapphire-hardhat
   ```
 
-4. Install `@nomicfoundation/hardhat-toolbox`, TypeScript and other peer
-   dependencies required by HardHat.
-
 ### Add the Sapphire Testnet to Hardhat
 
 Open up your `hardhat.config.ts` and drop in these lines.
 
-```diff
-diff --git a/hardhat.config.ts b/hardhat.config.ts
-index 414e974..49c95f9 100644
---- a/hardhat.config.ts
-+++ b/hardhat.config.ts
-@@ -1,8 +1,19 @@
+```js title="hardhat.config.ts"
  import { HardhatUserConfig } from "hardhat/config";
-+import '@oasisprotocol/sapphire-hardhat';
+ // highlight-next-line
+ import '@oasisprotocol/sapphire-hardhat';
  import "@nomicfoundation/hardhat-toolbox";
 
  const config: HardhatUserConfig = {
-   solidity: "0.8.17",
-+  networks: {
-+    'sapphire-testnet': {
-+      // This is Testnet! If you want Mainnet, add a new network config item.
-+      url: "https://testnet.sapphire.oasis.io",
-+      accounts: process.env.PRIVATE_KEY
-+        ? [process.env.PRIVATE_KEY]
-+        : [],
-+      chainId: 0x5aff,
-+    },
-+  },
+  solidity: "0.8.19",
```

Comment:
This should work fine.

```suggestion
  solidity: "0.8.24",
```

Related: https://github.com/rust-ethereum/evm/issues/277, https://github.com/oasisprotocol/oasis-sdk/issues/1853